pkgs/applications/window-managers/i3/lock-fancy.nix

```diff
@@ -43,12 +37,16 @@ stdenv.mkDerivation rec {
     cp i3lock-fancy $out/bin/i3lock-fancy
     ln -s $out/bin/i3lock-fancy $out/bin/i3lock
     cp icons/lock*.png $out/share/i3lock-fancy/icons
+
+    wrapProgram $out/bin/i3lock-fancy \
+      --prefix PATH : ${lib.makeBinPath [ coreutils fontconfig gawk getopt i3lock-color imagemagick scrot ]}
```

Comment:
Add `runHook {pre,post}Install`, and `installManPage doc/i3lock-fancy.1` from installShellFiles.